packages/client/test/end-to-end/resend.test.ts

```diff
@@ -16,18 +16,8 @@ describe('resend', () => {
     let resendClient: StreamrClient
 
     beforeEach(async () => {
-        publisherClient = new StreamrClient({
-            ...CONFIG_TEST,
-            auth: {
-                privateKey: await fetchPrivateKeyWithGas()
-            }
-        })
-        resendClient = new StreamrClient({
-            ...CONFIG_TEST,
-            auth: {
-                privateKey: fastPrivateKey()
-            }
-        })
+        publisherClient = createTestClient(await fetchPrivateKeyWithGas(), 'resend-e2e-publisher-client', 43232)
```

Comment:
Could we generate ids automatically? Maybe just an uuid. 

(It is good to have human-readable ids when we'd test network level connectivity, but this test and other Broker/client tests are testing different aspects. As connectivity issues are guarded with separate tests, we don't need to worry about that here.)

Reply:
The ids could be auto-generated but the set ids make debugging possible problems in the network layer easier.

Reply:
The ID's are now autogenerated if they are not explicitly given as a parameter.

Reply:
Great 👍 If we want to keep using explicit ids in end-to-end tests, we could use `getTestName` utility: https://github.com/streamr-dev/network/blob/149f437da8083e5c3529a142178e78b1b76b7c5a/packages/client/test/test-utils/utils.ts#L46